benchmarks/inc/utility.hpp

```diff
@@ -0,0 +1,47 @@
+#pragma once

```

Comment:
Missing license-and-copyright header. (Maybe put `xoshiro256ss` in a separate header just for CC0-1.0 code?)